.github/workflows/e2e.yml

```diff
@@ -32,27 +33,36 @@ concurrency:
 jobs:
   e2e:
     runs-on: ubuntu-latest
+
+    env:
+      AGORIC_NET: >-
+        ${{ github.event.inputs.network 
+            || contains(github.event.pull_request.labels.*.name, 'emerynet') && 'emerynet'
+            || contains(github.event.pull_request.labels.*.name, 'devnet') && 'devnet'
+            || contains(github.event.pull_request.labels.*.name, 'ollinet') && 'ollinet'
+            || contains(github.event.pull_request.labels.*.name, 'xnet') && 'xnet'
+            || github.event_name == 'schedule' && 'emerynet'
+            || 'local'
+          }}
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Run E2E Tests
         run: |
-          docker compose -f test/e2e/docker-compose.yml --profile synpress up --build --exit-code-from synpress
+          docker compose -f test/e2e/docker-compose.yml --profile $SYNPRESS_PROFILE up --build --exit-code-from synpress
         env:
+          A3P_IMAGE_TAG: ${{ secrets.IMAGE_TAG || 'latest' }}
```

Comment:
shouldnt this be an input rather than a secret?

Reply:
Thanks for pointing it out. It should be an input.